cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -6324,6 +6332,32 @@ def statistics(self):
         """
         return self.solver.getStatistics()
 
+    def unsat_core(self):
+        """Return a subset (as a list of Bool expressions) of the assumptions provided to the last check().
```

Comment:
plz run the `black` code formatter thing, I am not sure this is properly formatted.